domain/src/main/java/com/teamagam/gimelgimel/domain/map/SelectEntityInteractor.java

```diff
@@ -52,23 +44,7 @@ protected SelectEntityInteractor(
     @Override
     protected Iterable<SubscriptionRequest> buildSubscriptionRequests(
```

Comment:
In buildSelectEntityRequest it's assumed that the given ID will be successfully mapped to an entity.
I think it'll be best to check for it and log the cases where there's no entity mapped to the given ID.
[It's something I wrote if I'm not mistaking... my bad...]